backup.md

```diff
@@ -1,59 +1,247 @@
-# Backup setup notes
+# Backup and restore
 
-## From my workstation
+There are two bits of data that we spend effort backing up:
+
+* the montagu db
+* the orderly archive
+
+There are other bits of persistant data that we might want to expand this approach to, but these are currently less important:
+
+* the packit db (this will become important eventually)
+* the orderly logs
+* the packit redis data (not even persisted to a volume at present)
+
+There's a lot of historical cruft and faffage through here, much of which is related to the postgres database.  Once we start our journey of deprecating that, things will become much easier, and the backup/restore path for orderly will be the main concern.
+
+## Installation
+
+You will need to run this on any machine that you are running commands from
 
 ```
-privateer keygen --all
+pip3 install --user privateer~=2.1.0
 ```
 
-then
+Once installed, you can check the version by running
 
 ```
-vault list /secret/vimc/privateer
+privateer --version
 ```
 
+## Restore
+
+This is most often done to restore onto `uat` or `science`, refreshing the data that is stored there.
```

Comment:
So "For Dummies" - by "restore" we mean taking a schklump of data from a backup existence and pushing it into a functional Montagu server such as uat or science, (or in a recovery sort of situation, to production)